source/server-discovery-and-monitoring/server-monitoring.rst

```diff
@@ -697,7 +697,7 @@ The event API here is assumed to be like the standard `Python Event
                 topology.onServerDescriptionChanged(description, connection pool for server)
                 if description.error != Null:
                     # Clear the connection pool only after the server description is set to Unknown.
-                    clear connection pool for server
+                    clear(closeInUseConnections: isNetworkError(description.error)) connection pool for server
```

Comment:
isNetworkError -> isNetworkTimeout

isNetworkTimeout is also used in `source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst`

Reply:
done